src/test/java/org/jitsi/meet/test/DesktopSharingTest.java

```diff
@@ -37,128 +31,76 @@
     extends WebTestBase
 {
     /**
-     * A property to specified the external script that will be used to
-     * start the new participant.
+     * The id of the desktop sharing button.
      */
-    public final static String HOOK_SCRIPT = "desktop.sharing.hook.script";
+    private static String DS_BUTTON_ID = "toolbar_button_desktopsharing";
 
     /**
-     * Exception on starting the hook, null if script has started successfully.
+     * The XPATH of the desktop sharing button.
      */
-    private Exception hookException = null;
+    private static String DS_BUTTON_XPATH = "//a[@id='" + DS_BUTTON_ID + "']";
 
     @Override
-    public boolean skipTestByDefault()
+    public void setupClass()
     {
-        return true;
+        super.setupClass();
+
+        ensureOneParticipant();
+
+        ensureTwoParticipants(
+            null, null, null,
+            new WebParticipantOptions().setChromeExtensionId(
+                (String) MeetUtils.getConfigValue(
```

Comment:
Can we move getConfigValue method to WebParticipant ? Then you can call getParticipant1().getConfigValue(). Actually it could event be a method "getDesktopSharingExtensionId()" in the WebParticipantDirectly.

Reply:
The reason I ask for that is that we're trying to go away from accessing the driver directly and using it with static utility functions.

Reply:
See the previous comments. I've agreed with this and getConfigValue is already a moved to WebParticipant.